docs/api-reference/attribute-manager.md

```diff
@@ -60,6 +60,7 @@ Takes a single parameter as a map of attribute descriptor objects:
 
 * keys are attribute names
 * values are objects with attribute definitions:
+  + `type` (Enum, optional) - data type of the attribute, can be one of the following: `GL.FLOAT`, `GL.DOUBLE`, `GL.UNSIGNED_SHORT`, `GL.UNSIGNED_INT`, `GL.UNSIGNED_BYTE`, `GL.BYTE`, `GL.SHORT`, and `GL.INT`.
```

Comment:
I feel we need to describe the semantics of these constants, especially GL.DOUBLE, how they affect various input cases and modes. 

Especially as this doc would be a starting point for docs for the new "gpu-tables" module we discussed today.